src/js/stores/MesosStateStore.js

```diff
@@ -81,10 +81,11 @@ class MesosStateStore extends GetSetBaseStore {
       "/mesos/api/v1?get_master"
     )
       .retryWhen(linearBackoff(RETRY_DELAY, MAX_RETRIES))
-      .do(message => {
+      .map(({ response }) => response)
```

Comment:
`@dcos/mesos-client` still depends on `@dcos/http-service@0.3.0` maybe that's the issue?

Reply:
But this is `getMasterRequest` which should be a normal `request`, shouldn't it? Do you think two different versions are a problem?

Reply:
https://github.com/dcos/dcos-ui/pull/3216/files#diff-22eec13077e445e982cd1a9232667028R2

Reply:
Ah got it, it's also called request but from some other library 🙈